config/config.exs

```diff
@@ -65,11 +65,11 @@ config :release_notes_bot, confluence_email: System.get_env("CONFLUENCE_EMAIL")
 
 config :release_notes_bot, confluence_api_key: System.get_env("CONFLUENCE_API_KEY")
 
-config :release_notes_bot, client_secret: System.get_env("CLIENT_SECRET")
+# config :release_notes_bot, client_secret: System.get_env("CLIENT_SECRET")
```

Comment:
why are we commenting these out instead of just deleting them altogether?

Reply:
I think we may be revisiting OAuth in the near future, but for now OAuth is a feature that was started and never finished. I don't want to completely remove it right now, but it is an artifact. Artifacts are never good.